server/src/modules/user/user.controller.ts

```diff
@@ -112,4 +105,32 @@ export class UserController {
       throw new BadRequestException(error.response);
     }
   }
+
+  @PublicRoute()
+  @SuperAdmin()
```

Comment:
why do we need these two decorators? we can directly use `@UseGuards(SuperAdminAuthGuard)`

Reply:
implement `@UseGuards(SuperAdminAuthGuard)` as discussed earlier, and `@publicRoute()` decorator is required as AuthGuard is working globally thus its token checking should not work for `/admin` thus implemented